scripts/footprint/size_report

```diff
@@ -320,26 +336,32 @@ def print_tree(data, total, depth):
                 s = bcolors.WARNING + p[-1] + bcolors.ENDC
             else:
                 s = bcolors.OKBLUE + p[-1] + bcolors.ENDC
-            print('{:80s} {:20d} {:8.2f}%'.format("  "*(len(p)-1) + s, data[i], percent_c ))
+            print('{:80s} {:20d} {:8.2f}%'.format(
+                "  " * (len(p) - 1) + s, data[i], percent_c))
         else:
-            print('{:80s} {:20d} {:8.2f}%'.format(bcolors.OKBLUE + i + bcolors.ENDC, data[i], percent_c ))
+            print('{:80s} {:20d} {:8.2f}%'.format(
+                bcolors.OKBLUE + i + bcolors.ENDC, data[i], percent_c))
 
-    print('='*110)
+    print('=' * 110)
     print('{:92d}'.format(total))
     return totp
 
 
 binary = os.path.join(options.outdir, options.binary + ".elf")
 
 if options.outdir and os.path.exists(binary):
-    fp =  get_footprint_from_bin_and_statfile("%s/%s.bin" %(options.outdir, options.binary),
-            "%s/%s.stat" %(options.outdir,options.binary), 0, 0 )
+    fp = get_footprint_from_bin_and_statfile(
+        "%s/%s.bin" % (options.outdir, options.binary),
```

Comment:
Unrelated to the patch, but this should be os.path.join(), so that it works well under all OSes.